apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js

```diff
@@ -15,7 +20,7 @@ module.exports = superclass => class extends superclass {
       const postcode = req.sessionModel.get('property-address-postcode');
 
       if (inRange(tenancyStart)) {
-        req.sessionModel.set('valid-tenancy', isPilot(postcode));
+        req.sessionModel.set('valid-tenancy', isPilotOrUndefined(postcode));
```

Comment:
As mentioned above I could have put the if statement here but it just would have been harder to read